openedx/core/djangoapps/profile_images/views.py

```diff
@@ -47,16 +50,20 @@ class ProfileImageUploadView(APIView):
 
     **Response for POST**
 
-        Users can only upload their own profile image. If the requesting user does not have username
-        "username", this method will return with a status of 403 for staff access but a 404 for ordinary
-        users to avoid leaking the existence of the account.
+        If the requesting user tries to upload the image for a different user:
+
+        * If the requesting user has staff access, a 403 error is returned.
```

Comment:
Not sure what's doing the returning - the API? Would it be correct to say "If the requesting user has staff access, the API returns a 403 error"?